kubetest/main.go

```diff
@@ -480,6 +480,10 @@ func prepareGcp(o *options) error {
 			return fmt.Errorf("fail to set project %s : err %v", p, err)
 		}
 
+		if err := os.Setenv("PROJECT", p); err != nil {
```

Comment:
We will stop leaking state from one process to the next. We will pass state explicitly